### DISCUSSION TITLE
Host UnoWasm in .net core wep api project - ASP NET Core hosted

### DISCUSSION THREAD

**Discussion by e066377** [unoplatform/uno#6531] [Q&A]
In most cases a web assembly project needs a web api project of its own. In this case it will be preferable to have only one web site to serve both api and webassembly client.

What is needed is like Blazor core hosted model:
https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-5.0

Adding an option for core-hosted model in project templates that include wasm will be very helpful.

How can i achieve this result untill such an option is added to project templates?

**Comment by jeromelaban** [comment]
We do not have the documentation nor templates to build such a Web site at this point, we're tracking this in https://github.com/unoplatform/uno/discussions/6531.

Yet, [Static files serving](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-5.0) from an ASP.NET core application is generally a good way to handle this.
